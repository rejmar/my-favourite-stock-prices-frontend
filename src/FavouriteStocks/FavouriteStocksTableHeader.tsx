import classNames from "classnames";
import React, { FunctionComponent } from "react";

const FavouriteStocksTableHeader: FunctionComponent = () => {
  return (
    <thead
      data-testid="stocks-table__header"
      className={classNames("thead-dark")}
    >
      <tr className={classNames("d-flex")}>
        <th scope="col" className={classNames("col-6")}>
          Stock Name
        </th>
        <th scope="col" className={classNames("col-2")}>
          Timestamp
        </th>
        <th scope="col" className={classNames("col-2")}>
          Price
        </th>
        <th scope="col" className={classNames("col-2")}>
          Action
        </th>
      </tr>
    </thead>
  );
};

export default FavouriteStocksTableHeader;
