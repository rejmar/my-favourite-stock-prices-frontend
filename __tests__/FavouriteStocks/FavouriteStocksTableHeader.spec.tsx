import React from "react";
import renderer from "react-test-renderer";
import FavouriteStocksTableHeader from "../../src/FavouriteStocks/FavouriteStocksTableHeader";

describe("FavouriteStocksTableHeader tests", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<FavouriteStocksTableHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
