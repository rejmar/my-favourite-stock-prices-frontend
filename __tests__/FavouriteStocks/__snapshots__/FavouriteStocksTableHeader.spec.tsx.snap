// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavouriteStocksTableHeader tests matches snapshot 1`] = `
<thead
  className="thead-dark"
  data-testid="stocks-table__header"
>
  <tr
    className="d-flex"
  >
    <th
      className="col-6"
      scope="col"
    >
      Stock Name
    </th>
    <th
      className="col-2"
      scope="col"
    >
      Timestamp
    </th>
    <th
      className="col-2"
      scope="col"
    >
      Price
    </th>
    <th
      className="col-2"
      scope="col"
    >
      Action
    </th>
  </tr>
</thead>
`;
